<launch>
	<node name="sim" pkg="glider_kayak_sim" type="ross_kayak.py" output="screen"/>
	<node name="sensor" pkg="glider_kayak_sim" type="kayak_sensor.py" output="screen"/>
	<node name="waypoint_chatter" pkg="glider_kayak_sim" type="waypoint_chatter.py"/>
</launch>
